lib/staging/slac/fsm/evse/include/slac/fsm/evse/context.hpp

```diff
@@ -212,6 +213,7 @@ struct Context {
     void log_info(const std::string& text);
 
     ModemVendor modem_vendor{ModemVendor::Unknown};
+    slac::messages::cm_slac_match_cnf match_cnf_message;
```

Comment:
This doesn't really belong here, as it is shared information only between Matching and WaitForLink state.  Matching state should keep it for itself and pass it to WaitForLink state with `create_simple<WaitForLink>(ctx, std::move(match_cnf_message))`.  You would need to modify the constructor of WaitForLink state.